cppForSwig/BlockDataMap.h

```diff
@@ -48,11 +48,16 @@ struct BCTX
    const uint8_t* data_;
    const size_t size_;
 
+   BinaryData noWitData_;
```

Comment:
make this local during hash computation, no need to carry that data beyond that one method